action.yaml

```diff
@@ -197,6 +209,15 @@ runs:
             echo "    File ${{ inputs.prometheus_config }} not found. Using defaults instead."
         fi
 
+    - name: Set data retention and scrape interval
+      shell: bash
+      run: |
+        echo "Setting retention days and scrape_interval"
+        yq eval -i '.global.scrape_interval = "${{ inputs.prometheus_scrape_interval }}"'  "${{ github.action_path }}/observability/prometheus/prometheus.yml"
+        yq eval -i '.global.evaluation_interval = "${{ inputs.prometheus_scrape_interval }}"'  "${{ github.action_path }}/observability/prometheus/prometheus.yml"
+        yq eval -i '.datasources[0].timeInterval += "${{ inputs.grafana_scrape_interval }}"'  "${{ github.action_path }}/observability/grafana/datasources/sources.yml"
+        yq eval -i '.services.prometheus.command += ["--storage.tsdb.retention.time=${{ inputs.prometheus_retention_period }}"]' "${{ github.action_path }}/docker-compose.yaml"
```

Comment:
if the user provides their own `prometheus.yml` or `sources.yml` files, those files will take precedence over the GHA input values, right @LeoDiazL ?

Reply:
It will apply to either. I will move it to only apply when using the default files instead. 
For the prometheus part, it will apply to the global settings. If someone defines something different below, it will override it. 
For the grafana part, it will apply to the first datasource only.  (Will only make sense if we are setting this up just for the default files)